.github/workflows/lint-secrets.yml

```
@@ -31,6 +31,8 @@ jobs:
           rm -rf .tmp
       - name: Configure git-secrets
         run: |
+          # workaround git-secrets requiring the say command
+          sudo ln -s "$(which echo)" /usr/local/bin/say
```

The action already runs as root. :)
```suggestion
          ln -s "$(which echo)" /usr/local/bin/say
```

---

you've never heard of super-root?